test/disabled_test_memory_leak_on_error.js

```diff
@@ -1,31 +1,37 @@
 // See: https://github.com/kripken/sql.js/issues/306
-exports.test = function(sql, assert) {
-  var errors = 0, runs=10000;
-  for (var i=0; i<runs; i++) {
-    var db = new sql.Database()
-    try {
-      db.exec("CREATE TABLE cats (name TEXT NOT NULL, age INTEGER NULL)")
-      db.exec("INSERT INTO cats (name, age) VALUES (NULL, 3)")
-    } catch (e) {
-      errors += e.toString() === "Error: NOT NULL constraint failed: cats.name";
+
+"use strict";
+
+exports.test = function (sql, assert) {
+    var errors = 0; var
+        runs = 10000;
```

Comment:
?

Reply:
sorry will fix ^^;;  it was actually fixed in commit 4 (before reverting).

i suspect its an eslint glitch when i autofix whitespace with command `npm run prettify`

Reply:
fixed in commit 4631779f:
verified no more of that silliness with shell-command:
```shell
git grep -i '; var'
# src/worker.js:    var buff; var data; var result;
```